Travis/js/main.js

```diff
@@ -0,0 +1,219 @@
+
+
+
+
+// A light switch that can be either on or off. Booleans
+
+let light = false;
+
+// A user's email address. Objects  
+ 
+let email = { 
+	name: 'unknown',
+	email: 'email@unknown.com'
+}
+
+// A spaceship with a hull, laser blasters, tractor beam, and warp drive. String
+
+let spaceship = ('hull', 'laser blasters', 'tractor beam', 'warp drive')
+
+
+// A list of student names from our class. Strings
+let wdi10 = ('kait', 'charls', 'michael')
```

Comment:
a list would be an array. more or less same issues as above (ie also the syntax thing)
-0.75